examples/all-clusters-minimal-app/esp32/main/DeviceWithDisplay.cpp

```diff
@@ -450,11 +456,21 @@ class SetupListModel : public TouchesMatterStackModel
             commissionMgr.OpenBasicCommissioningWindow(commissionMgr.MaxCommissioningTimeout(),
                                                        CommissioningWindowAdvertisement::kDnssdOnly);
         }
+        else if (i == 3)
+        {
+            ESP_LOGI(TAG, "START DELAY");
+            for (int i = 0; i < 1000; ++i)
```

Comment:
why the loop and not vTaskDelay(1000) ? Also these are ticks. Should we use `portTICK_PERIOD_MS` to make this actually visible as a standard time?

Reply:
removed these changes as they are not required

Reply:
removed these changes as they are not required